cmd/rbenv-sync.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require "cli/parser"
+require "formula"
+
+module Homebrew
+  module_function
+
+  def rbenv_sync_args
+    Homebrew::CLI::Parser.new do
+      usage_banner <<~EOS
+        `rbenv-sync`
+
+        Create symlinks for Homebrew's installed NodeJS versions in ~/.nodenv/versions.
+
+        Note that older version symlinks will also be created so e.g. Ruby 3.2.1 will
+        also be symlinked to 3.2.0.
```

Comment:
Not sure I understand what this means.

Reply:
@carlocab this may help:

```
mikebook # ls -lha ~/.rbenv/versions 
Permissions Size User Group Date Modified Git Name
lrwxr-xr-x    37 mike staff 14 Mar 06:30   -- 3.1.0 -> /opt/homebrew/Cellar/ruby@3.1/3.1.3_1/
lrwxr-xr-x    37 mike staff 14 Mar 06:30   -- 3.1.1 -> /opt/homebrew/Cellar/ruby@3.1/3.1.3_1/
lrwxr-xr-x    37 mike staff 14 Mar 06:30   -- 3.1.2 -> /opt/homebrew/Cellar/ruby@3.1/3.1.3_1/
lrwxr-xr-x    37 mike staff 14 Mar 06:30   -- 3.1.3 -> /opt/homebrew/Cellar/ruby@3.1/3.1.3_1/
lrwxr-xr-x    31 mike staff 14 Mar 06:30   -- 3.2.0 -> /opt/homebrew/Cellar/ruby/3.2.1/
lrwxr-xr-x    31 mike staff 14 Mar 06:30   -- 3.2.1 -> /opt/homebrew/Cellar/ruby/3.2.1/
```

Make more sense?

This is because we don't ship individual patch versions and, in my experience working with these tools at scale for a non-trivial amount of time, having differing patch versions in development makes zero difference if you have any sort of meaningful CI/production verification.